python/fusion_config/synonyms.txt

```diff
@@ -0,0 +1 @@
+ceo, will hayes
```

Comment:
Yep, just figured I'd commit in the framework for synonyms first, then start attacking specific relevancy issues.